### PR TITLE
[org.eclipse.ui.workbench.texteditor.tests]: Add @After methods to tear down the created Composites

### PR DESCRIPTION
**Pull request by Wittmaxi**
This prevents a leak of OS-resources which cannot be reached by the JVM-GarbageCollector

### CODE REVIEW

File: tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/workbench/texteditor/tests/TextViewerDeleteLineTargetTest.java [hunk 1]
```diff
@@ -46,12 +49,25 @@ public void setUp() {
...
 	}
 
+	@After
+	public void tearDown() {
+		if (parentShell != null) {
+			parentShell.dispose();
+			parentShell= null;
+		}
+		if (textViewer != null) {
+			textViewer.getControl().dispose();
```
Comment (mickaelistria) [inline tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/workbench/texteditor/tests/TextViewerDeleteLineTargetTest.java]: As textViewer.getControl() is a child of parentShell, this seems useless too.

**Review by mickaelistria — APPROVED** [review]
Thank you. We'll merge as soon as CI succeeds.

File: tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/workbench/texteditor/tests/TextViewerDeleteLineTargetTest.java [hunk 2]
```diff
@@ -38,6 +39,8 @@ public class TextViewerDeleteLineTargetTest {
 
 	private Document document;
 
+	private Shell parentShell;
+	private TextViewer textViewer;
```
Comment (mickaelistria) [inline tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/workbench/texteditor/tests/TextViewerDeleteLineTargetTest.java]: You actually don't need to make it a field here then.

File: tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/workbench/texteditor/tests/minimap/MinimapWidgetTest.java [hunk 3]
```diff
@@ -46,22 +47,34 @@ public class MinimapWidgetTest {
 
 	private ITextViewer editorViewer;
 
+	private Composite minimapParent;
+
+	private MinimapWidget minimapWidget;
```
Comment (mickaelistria) [inline tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/workbench/texteditor/tests/minimap/MinimapWidgetTest.java]: You actually don't need to make it a field here then.

**Review by HeikoKlare — APPROVED** [review]
Changes look good now. Test failures are unrelated and already documented: #275, #926, #1183

**Pull request merged**